baseplate/metrics.py

```diff
@@ -193,8 +194,31 @@ def __exit__(self, exc_type, value, traceback):
 
     def flush(self):
         """Immediately send the batched metrics."""
+        for name, counter in self.counters.items():
+            counter.send()
         self.transport.flush()
 
+    def counter(self, name):
+        """Return a BatchCounter with the given name.
+
+        BatchCounter instances are stored in the :term:`counters` attribute.
+        They are reused on subsequent calls.
+
+        The sample rate is currently up to your application to enforce.
+
+        :param str name: The name the counter should have.
+
+        :rtype: :py:class:`BatchCounter`
```

Comment:
This subclass isn't publicly documented (and that's good, I think, since it meets the LSP and is just an internal detail) and so the link doesn't work in docs. Maybe just say `:py:class:`Counter`` here? It's not a lie since a BatchCounter is a Counter! :)